04_containers.Rmd

```diff
@@ -362,21 +362,15 @@ seqtab.nochim <- readRDS("data/dada2_seqtab.nochim")
 taxa <- readRDS("data/dada2_taxa")
 ```
 
-Load required packages.
+Load libraryd packages.
```

Comment:
This is error. The textual explanations should remain "Load required packages". Only the R command was changed from "require" to "library".